### PR TITLE
Bug 1195268

### PR DESCRIPTION
**Pull request by Ritsyy**
Removed flickering from screenshot arrows(bug 1195268)

### CODE REVIEW

**Comment by atiqueahmedziad** [comment]
r ? @ngokevin

File: src/media/js/previews-buttons.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ define('previews-buttons',
 
         $container.find('.arrow-btn-prev, .arrow-btn-next').remove();
 
-        var $prevBtn = $('<a class="arrow-btn arrow-btn-prev"></a>');
-        var $nextBtn = $('<a class="arrow-btn arrow-btn-next"></a>');
+        var $prevBtn = $('<a class="arrow-btn arrow-btn-prev" data-no-transparent="true"></a>');
+        var $nextBtn = $('<a class="arrow-btn arrow-btn-next" data-no-transparent="true"></a>');
```
Comment (chuckharmston) [inline src/media/js/previews-buttons.js]: I don't think we need `="true"`, just the presence of the attribute name should be enough, e.g.

https://github.com/mozilla/fireplace/blob/ac3ea0654011c1fca4387c9fcb2693076f1d525d/src/templates/games/index.html#L40

**Comment by atiqueahmedziad** [comment]
Thanks, for the changes :)
Please, squash the commits into one 
use the command- 
`git rebase -i HEAD~2`
pick the first commit & squash the second one
the give
`git push-f`

**Comment by atiqueahmedziad** [comment]
Thanks :) 
r ? @chuckharmston 
can you merge it ?

**Comment by ngokevin** [comment]
this bug might not apply anymore, i might hold off on this until i look into button transparency bugs more.

**Comment by ngokevin** [comment]
dependent on https://github.com/mozilla/fireplace/pull/1474/files

**Comment by ngokevin** [comment]
i removed the data-no-transparent thing...though it might still be reproducing. you can reopen if the issue still exists and then we need a different solution. thanks!

**Comment by waseem18** [comment]
@ngokevin @atiqueahmedziad  I guess the bug is not yet resolved. I can still find the flickering.

**Pull request closed without merging**